cli/build/build_test.go

```diff
@@ -24,17 +25,31 @@ func TestFillCtx(t *testing.T) {
 	require.NoError(t, os.Chdir(workDir))
 	defer os.Chdir(wd)
 	var buildCtx BuildCtx
+
+	// Output of os.getwd after chdir and t.tempdir is
+	// different on macOS ("/private/var" and "/var/"),
+	// but it is the same directory, since "/var" is a symlink to "/private/var"
+	// and it should not cause test to fail.
+	os, err := util.GetOs()
+	require.NoError(t, err)
+	if util.OsMacos == os {
+		workDir = filepath.Join("/private", workDir)
+	}
+
+	appDir := filepath.Join(workDir, "app1")
+	appDir2 := filepath.Join(workDir, "app2")
+
 	require.NoError(t, FillCtx(&buildCtx, []string{"app1"}))
-	assert.Equal(t, buildCtx.BuildDir, filepath.Join(workDir, "app1"))
+	assert.Equal(t, buildCtx.BuildDir, appDir)
```

Comment:
I don't like the idea to rely on specific symlinks target and hard-coded directory names like "/private".
Let's just check the suffix here (strings.HasSuffix workdir + app1). Or use os.Getwd result instead of tempDir for file names comparison.

Reply:
Changed to os.Getwd file names comparison